### PR TITLE
docs: open multiple terminals side-by-side

### PR DESCRIPTION
**Pull request by baobaoit**
This PR explains issue #93.

### CODE REVIEW

**Comment by baobaoit** [comment]
Hello @akinsho , please review my PR. Thank you.

File: README.md [hunk 1]
```diff
@@ -351,3 +351,25 @@ You can create your on commands by using the lua functions this plugin provides
 command! -count=1 TermGitPush  lua require'toggleterm'.exec("git push",    <count>, 12)
 command! -count=1 TermGitPushF lua require'toggleterm'.exec("git push -f", <count>, 12)
 ```
+
+### Open multiple terminals side-by-side
+| Direction | Tested |
+| --- | --- |
+| vertical | ✔️ |
+| horizontal | ✔️ |
+| window | ⚠️ |
```
Comment (akinsho) [inline README.md:360]: These are unsupported by definition e.g. a window is supposed to occupy the whole screen so by definition you can show multiple window mode terms side by side, also this is unsupported for floats I would just put an `X` there to explain that it isn't supported

File: README.md [hunk 2]
```diff
@@ -351,3 +351,25 @@ You can create your on commands by using the lua functions this plugin provides
...
+| float | ⚠️ |
+
+✔️: tested, working.
+
+⚠️: not tested, may not working.
+
+On your first terminal, you need to escape the `TERMINAL` mode via the `<Esc>` key that has been mapped with the combination `<C-\><C-n>`.
```
Comment (akinsho) [inline README.md:367]: This is a minor nit but you can improve the rendering of a keyboard character using `<kbd>Esc</kbd>` which gives <kbd>Esc</kbd>, or `<kbd>C-\\</kbd><kbd>C-N</kbd>` which gives <kbd>C-\\</kbd><kbd>C-N</kbd>

Comment (akinsho) [inline README.md:367]: I would say 
```suggestion
In your first terminal, you need to leave `TERMINAL` mode using `<C-\><C-n>` which can be remapped to `Esc` for ease of use.
```

File: README.md [hunk 3]
```diff
@@ -351,3 +351,25 @@ You can create your on commands by using the lua functions this plugin provides
...
+![image](https://user-images.githubusercontent.com/31947091/133395516-22fef1e6-633d-4964-9175-f76fabf66794.png)
+
+Then you type on: `2<C-\>`, and the result:
+![image](https://user-images.githubusercontent.com/31947091/133396789-fdf68b30-3a8c-440b-822f-6549f282c4fc.png)
+
+Explain:
+- `2`: this is the terminal identity, your first terminal is `1` (e.g. your 3rd terminal will be `3<C-\>`, so on).
```
Comment (akinsho) [inline README.md:374]:
```suggestion
- `2`: this is the terminal's number (or ID), your first terminal is `1` (e.g. your 3rd terminal will be `3<C-\>`, so on).
```

**Review by akinsho — CHANGES_REQUESTED** [review]
@baobaoit thanks so much for the PR. I've added some comments about some improvements, some grammatical things/changes in phrasing etc. 🙏🏿

**Comment by baobaoit** [comment]
@akinsho thanks for the advice. Please let me know if it needs improvement any things else.

**Comment by akinsho** [comment]
Awesome thanks for making those tweaks 👍🏿 this LGTM now 🚀

**Pull request merged**